packages/components/card/examples/Card/WithCustomDragHandle.tsx

```diff
@@ -1,76 +1,82 @@
 import React from 'react';
 import { Box, Card, DragHandle, Flex } from '@contentful/f36-components';
 import { css } from 'emotion';
+import { DndContext } from '@dnd-kit/core';
 import {
-  SortableContainer,
-  SortableElement,
-  SortableHandle,
-} from 'react-sortable-hoc';
-import { arrayMoveImmutable } from 'array-move';
+  arrayMove,
+  verticalListSortingStrategy,
+  SortableContext,
+  useSortable,
+} from '@dnd-kit/sortable';
+import { CSS } from '@dnd-kit/utilities';
 
 export default function WithCustomDragHandle() {
   const styles = {
     dragHandle: css({
       alignSelf: 'stretch',
     }),
   };
-  const [items, updateItems] = React.useState([
+  const [items, setItems] = React.useState([
     'Tech',
     'News',
     'CMS',
     'Contentful',
   ]);
 
-  const swapItems = React.useCallback(
-    ({ oldIndex, newIndex }) => {
-      // `arrayMoveImmutable` is imported from 'array-move'
-      const newItems = arrayMoveImmutable(items, oldIndex, newIndex);
-      updateItems(newItems);
-    },
-    [items],
-  );
-
-  const SortableList = SortableContainer((props) => (
-    <Flex flexDirection="column">{props.children}</Flex>
-  ));
+  function SortableCard({ id }) {
+    const { attributes, listeners, setNodeRef, transform, transition } =
+      useSortable({
+        id,
+      });
 
-  const SortableDragHandle = SortableHandle(() => (
-    <DragHandle as="button" className={styles.dragHandle} label="Move card" />
-  ));
+    const style = {
+      transform: CSS.Translate.toString(transform),
+      transition,
```

Comment:
Should it also have the z-index logic to show the dragged element always on top?

Reply:
I added this to the table example.